### PR TITLE
Shade/relocate picocli library.

### PR DESCRIPTION
**Pull request by gierlachg**
Properly shaded/relocated `picocli` library

Checklist
- [x] Tags Set
- [x] Milestone Set

### CODE REVIEW

File: hazelcast-jet-all/pom.xml [hunk 1]
```diff
@@ -67,6 +67,25 @@
...
                     <createDependencyReducedPom>true</createDependencyReducedPom>
+                    <artifactSet>
+                        <includes>
+                            <include>com.hazelcast.jet:hazelcast-jet-core</include>
+                            <include>com.hazelcast:hazelcast</include>
+                            <include>com.hazelcast:hazelcast-aws</include>
+                            <include>com.hazelcast:hazelcast-gcp</include>
+                            <include>com.hazelcast:hazelcast-kubernetes</include>
+                            <include>org.apache.geronimo.specs:geronimo-jms_1.1_spec</include>
+                            <include>com.rabbitmq.jms:rabbitmq-jms</include>
```
Comment (cangencer) [inline hazelcast-jet-all/pom.xml]: why are these here as shaded?

Comment (gierlachg) [reply hazelcast-jet-all/pom.xml]: These were already shaded implicitly, I have made it explicit.

Comment (cangencer) [reply hazelcast-jet-all/pom.xml]: I realized they're there because of a mistake in the core pom. Should be removed from there as well

File: hazelcast-jet-all/pom.xml [hunk 2]
```diff
@@ -67,6 +67,25 @@
                 <configuration>
                     <createSourcesJar>true</createSourcesJar>
                     <createDependencyReducedPom>true</createDependencyReducedPom>
+                    <artifactSet>
+                        <includes>
+                            <include>com.hazelcast.jet:hazelcast-jet-core</include>
```
Comment (cangencer) [inline hazelcast-jet-all/pom.xml]: is it possible to use wildcards here?
